### PR TITLE
Library updates, common code cleanup, other stuff

### PR DESCRIPTION
**Pull request by TacoTheDank**
This PR is an amalgamation of different things I found that needed cleaning in one way or another.

### CODE REVIEW

**Review by ccrama — COMMENTED** [review]
Looks great! Lots of code savings, thank you for the PR

File: app/src/main/java/me/ccrama/redditslide/Tumblr/Example.java [hunk 1]
```diff
@@ -1,79 +0,0 @@
-
-package me.ccrama.redditslide.Tumblr;
```
[the remark this answers is not in the document]
Comment (TacoTheDank) [reply app/src/main/java/me/ccrama/redditslide/Tumblr/Example.java:2]: Lol honestly, I don't know

File: app/.gitignore [hunk 2]
```diff
@@ -1 +0,0 @@
-/build
```
[the remark this answers is not in the document]
Comment (TacoTheDank) [reply app/.gitignore:1]: The root .gitignore already covered this case so we didn't need this.

File: .gitignore [hunk 3]
```diff
@@ -12,8 +12,7 @@ local.properties
 .signing/
 
 # User-specific configurations
-.idea/*
-!.idea/codeStyleSettings.xml
```
[the remark this answers is not in the document]
Comment (TacoTheDank) [reply .gitignore:16]: Well, we don't have `.idea/codeStyleSettings.xml` anyways, so I think it's pointless to have it listed here as kept.

File: app/src/main/java/me/ccrama/redditslide/Tumblr/Example.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (TacoTheDank) [reply app/src/main/java/me/ccrama/redditslide/Tumblr/Example.java:2]: I just know that we didn't need the Example class.

**Pull request merged**